platform-api/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/server/MachineService.java

```diff
@@ -331,6 +334,24 @@ public void getProcessLogs(@PathParam("machineId") String machineId,
         addLogsToResponse(machineManager.getProcessLogReader(machineId, pid), httpServletResponse);
     }
 
+    @GET
+    @Path("/{machineId}/filepath/{path:.*}")
+    @Produces(MediaType.TEXT_PLAIN)
+    @RolesAllowed("user")
+    public String getFileContent(@PathParam("machineId") String machineId,
+                                 @PathParam("path") String path)
+            throws NotFoundException, ForbiddenException, ServerException, IOException {
+        checkCurrentUserPermissions(machineManager.getMachine(machineId));
+
+        CommandDescriptor descriptor = newDto(CommandDescriptor.class)
+                .withCommandLine(String.format("cat %1$2s", path));
```

Comment:
i think you can define it in one line and without of external formatting 
<code> CommandDescriptor descriptor = newDto(CommandDescriptor.class).withCommandLine("cat " + path)</code>